src/store/actions/updateCommunity.ts

```diff
@@ -18,7 +18,7 @@ export interface Payload {
     groupExpressionData: GroupExpressionData
 }
 
-async function updateCommunity({commit, getters}: Context, {community, groupExpressionData}: Payload) {
+export default async function updateCommunity({commit, getters}: Context, {community, groupExpressionData}: Payload) {
     const groupExpressionLang = community.typedExpressionLanguages.find((val: JuntoExpressionReference) => val.expressionType == ExpressionTypes.GroupExpression);
```

Comment:
@jdeepee I would probably use the getter to get the community just based on communityId. Then when you dispatch the actions from somewhere in the UI, you only need a reference to the id, and not pass in the whole community object.